include/seqan3/std/concepts

```diff
@@ -17,6 +17,66 @@
 
 #ifdef __cpp_lib_concepts        // C++20 ranges available
 #include <concepts>
+
+namespace std
+{
+inline namespace deprecated
+{
+/*!\brief This should be named default_initializable.
+ * \deprecated Use std::default_initializable instead.
+ */
+template<class T>
+concept default_constructible = default_initializable<T>;
+
+/*!\brief This should be removed.
+ * \deprecated No viable alternative.
+ */
+template <class B>
+concept boolean =
```

Comment:
Can the concept boolean in line 133 be removed now since we define it here?

Reply:
It should also be removed in the future;

but we can't do it right now, because neither ranges-v3 (= non `-std=c++2a` mode) nor std::ranges (= `-std=c++2a` mode) defines it, but we use it in the library.